### PR TITLE
Bug/ui fixes

### PR DESCRIPTION
**Pull request by OlegGordiichuk**
- Remove of additional constraints.
- Clean code.
- Remove warnings.

### CODE REVIEW

File: Trust/Accounts/Views/AccountViewCell.swift [hunk 1]
```diff
@@ -8,14 +8,17 @@ protocol AccountViewCellDelegate: class {
 }
 
 class AccountViewCell: UITableViewCell {
+
```
Comment (vikmeup) [inline Trust/Accounts/Views/AccountViewCell.swift]: is this spacing intended?

File: Trust/Tokens/ViewControllers/EditTokensViewController.swift [hunk 2]
```diff
@@ -21,12 +21,12 @@ class EditTokensViewController: UITableViewController {
         let searchController = UISearchController(searchResultsController: self.searchResultsController)
         searchController.delegate = self
         searchController.searchResultsUpdater = self
+        searchController.hidesNavigationBarDuringPresentation = true
```
Comment (vikmeup) [inline Trust/Tokens/ViewControllers/EditTokensViewController.swift]: would this break anything else?

File: Trust/Tokens/ViewModels/NonFungibleTokenViewModel.swift [hunk 3]
```diff
@@ -37,6 +37,10 @@ class NonFungibleTokenViewModel {
         return !tokens.isEmpty
     }
 
+    var cellHeight: CGFloat {
+        return 263
```
Comment (vikmeup) [inline Trust/Tokens/ViewModels/NonFungibleTokenViewModel.swift]: Is it intended change? it was 250 points

File: Trust/Accounts/Views/AccountViewCell.swift (hunk 1 above)
Comment (OlegGordiichuk) [reply Trust/Accounts/Views/AccountViewCell.swift]: Unfortunately in this class we have issue with spacings. So yes.

File: Trust/Tokens/ViewControllers/EditTokensViewController.swift (hunk 2 above)
Comment (OlegGordiichuk) [reply Trust/Tokens/ViewControllers/EditTokensViewController.swift]: Nop, it is safe.

File: Trust/Tokens/ViewModels/NonFungibleTokenViewModel.swift (hunk 3 above)
Comment (OlegGordiichuk) [reply Trust/Tokens/ViewModels/NonFungibleTokenViewModel.swift]: Yeah but we add custom offsets and bounds. And in this case to give an option to build proper cell size and silent warning we should set this height.

Comment (vikmeup) [reply Trust/Tokens/ViewModels/NonFungibleTokenViewModel.swift]: Can you reduce to 250? or even 240, 263 seems too big

Comment (OlegGordiichuk) [reply Trust/Tokens/ViewModels/NonFungibleTokenViewModel.swift]: ok

**Pull request merged**